base/applications/taskmgr/lang/fr-FR.rc

```diff
@@ -1,4 +1,4 @@
-// COPYRIGHT: Benoit Plante (aka cmoibenlepro)
+// COPYRIGHT: Benoit Plante (cmoibenlepro)
```

Comment:
Unrelated change.

Reply:
Unrelated and shorter: yes.
Wrong: no.

Reply:
I'd prefer such changes in a separate PR and throughout entire source tree, fixing all occurences, i.e. not just because being "while at it"./

Reply:
There is just one additional hit in the whole source-tree, if you want you can create a PR to remove it there as well:
https://git.reactos.org/?p=reactos.git&a=search&h=HEAD&st=grep&s=aka+cmoibenlepro%29
I don't want it to be part of this PR, and I don't want to exclude my change from this PR neither.

Reply:
What I meant is the `"%Full_Name% (aka %nickname%)"` pattern.

Reply:
Like I said: Feel free to open a PR for it. But please stop nitpicking in mine about that comment-change. I will not undo that change.